dsv-rp/DDS
# DDS-917 Add toggle component

This PR adds Toggle (`daikin-toggle`) component.

## Related Issue(s) and Links

- JIRA Ticket (parent): [DDS-917](https://daikinsvoil.atlassian.net/browse/DDS-917)
- Figma design: [Figma](https://www.figma.com/design/VyaaU8Ta9yzyf0PsURWSSf/branch/wHAhw4yC54Hjq6Whb8FL5T/DDS%3A-Design-Kit?node-id=23902-559&m=dev)

## API Design
[JIRA Wiki](https://daikinsvoil.atlassian.net/wiki/spaces/GDS/pages/3097624601/Toggle+Component+API)

## How Has This Been Tested?

- VRT (Visual Regression Test) for [default, small] size and ["enabled", "disabled",] variant and ["checked", "unchecked",] state.
- Toggle will be checked if click on it and will be unchecked click again
- Toggle's check state will not be changed when disabled and click change event will not be fired

## Checklist

- [x] I have added appropriate VRTs.
- [x] I have added appropriate unit tests.
- [x] I have confirmed that my changes do not generate new warnings and errors.


[DDS-917]: https://daikinsvoil.atlassian.net/browse/DDS-917?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
